broker/src/main/fabric8/deployment.yml

```diff
@@ -8,10 +8,10 @@ spec:
           periodSeconds: 5
           timeoutSeconds: 1
           tcpSocket:
-            port: 5672
+            port: 5671
```

Comment:
We don't have a proper readiness probe in Artemis that's exposed using REST/JMX (I added it to the todo list). I think the best we can do at the moment is to check AMQP port, until we add readiness feature.